deploy/install/01-prerequisite/03-crd.yaml

```diff
@@ -73,6 +125,22 @@ spec:
     singular: volume
   scope: Namespaced
   version: v1beta1
+  additionalPrinterColumns:
+  - name: Size
```

Comment:
Add `Robustness`, and condition `Scheduled`.

Reply:
Thank you! Added